ring_doorbell/doorbot.py

```diff
@@ -289,16 +289,25 @@ def live_streaming_json(self):
                 pass
         return None
 
-    def recording_download(self, recording_id, filename=None, override=False):
+    def recording_download(self, recording_id, filename=None,
+                           override=False, timeout=None):
```

Comment:
Go ahead and initialize it directly instead of later. The user will be able to override it regardless if they want.
```python
    def recording_download(self, recording_id, filename=None,
                           override=False, timeout=DEFAULT_VIDEO_DOWNLOAD_TIMEOUT):
```

Reply:
Yeah, excellent point!  Pushing a commit momentarily.